src/java/org/apache/cassandra/locator/AbstractReplicationStrategy.java

```diff
@@ -20,9 +20,11 @@
 import java.lang.reflect.Constructor;
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.net.InetAddress;
```

Comment:
InetAddress and Set don't seem to be used in the new code?

Reply:
done